app/modules/asset_groups/models.py

```diff
@@ -128,6 +129,42 @@ class AssetGroupSighting(db.Model, HoustonModel):
     # May have multiple jobs outstanding, store as Json obj uuid_str is key, In_progress Bool is value
     jobs = db.Column(db.JSON, default=lambda: {}, nullable=True)
 
+    curation_start = db.Column(
+        db.DateTime, index=True, default=datetime.utcnow, nullable=False
+    )
+
+    def __init__(self, asset_group, sighting_config, detection_configs):
```

Comment:
Logic moved from Asset_group begin_ia_pipeline as it was all AGS specific. Started off being vrelated to change but ended up just being moved as it was in the wrong place.